content/en/docs/ambient/getting-started/snips.sh

```diff
@@ -59,6 +59,10 @@ kubectl apply -f samples/bookinfo/gateway-api/bookinfo-gateway.yaml
 }
 
 snip_deploy_the_sample_application_4() {
+kubectl annotate gateway bookinfo-gateway networking.istio.io/service-type=ClusterIP --namespace=default
+}
+
+snip_deploy_the_sample_application_5() {
```

Comment:
You created a new snip, test is failing because the exports from here aren't available when snip_verify_traffic_sleep_to_ingress is called in test.sh. You'll need to call _5 as well.